les/peer.go

```diff
@@ -94,6 +94,8 @@ type peer struct {
 	sendQueue *execQueue
 
 	errCh chan error
+	wg    sync.WaitGroup // Wait group used to track all in-flight task routines.
```

Comment:
Actually this does not need to be a part of the peer struct because it is only used in the handler.